### ISSUE TITLE
Bundler version command + version autoswitching = deceptive

### ISSUE THREAD

**Issue by ZimbiX** [bundler/bundler#6908]
Given I have Bundler 1.x and 2.x installed, and a `Gemfile.lock` specifying 1.x, when I run `bundler --version` in my project directory, then I expect to see that I have Bundler 2.x installed. Instead, it shows 1.x.

This is deceptive because different versions are shown when installing Bundler then checking its version:

```
$ gem install bundler
Successfully installed bundler-2.0.1
1 gem installed
$ bundler --version
Bundler version 1.17.1
```

And given it depends on the directory I am in, in a different one, I might see:

```
$ bundler --version
Bundler version 2.0.1
```

I take it this is due to the new Bundler version autoswitching.

I think for the version command, Bundler should not autoswitch. And when a `Gemfile.lock` specifies 1.x, it would be helpful to instead see something like:

```
$ bundler --version
Bundler version 2.0.1 (but using 1.17.1 because a 1.x version is specified in Gemfile.lock)
```

**Comment by duckinator** [comment]
I agree that this gets misleading sometimes. However, I think it makes sense to show which version is actually being run in that directory.

Another approach could be something like this for projects needing Bundler 1.x:

```shell
$ cd ~/some-bundler1-project
~/some-bundler1-project$ bundler --version
Bundler version 1.17.1
(Bundler version 2.0.1 is also available)
```

And then something like this for projects needing Bundler 2.x:
```
~/some-bundler1-project$ cd ~/some-bundler2-project
~/some-bundler2-project$ bundler --version
Bundler version 2.0.1
(Bundler version 1.17.1 is also available)
```

What are your thoughts, @ZimbiX?
